Src/LiquidProjections/IEventStoreWithSubscriptionIds.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LiquidProjections
+{
+    public interface IEventStoreWithSubscriptionIds : IEventStore
+    {
+        IDisposable Subscribe(long? lastProcessedCheckpoint, Func<IReadOnlyList<Transaction>, Task> handler,

```

Comment:
I suspect you introduced this extra interface to ensure backwards-compatibility?